inheritance-es5.js

```diff
@@ -0,0 +1,39 @@
+function Car (name, model, year, color, maxSpeed, fuelCapacity, fuelConsumption) {
+    this.name = name;
+    this.model = model;
+    this.year = year;
+    this.color = color;
+    this.maxSpeed = maxSpeed;
+    this.fuelCapacity = fuelCapacity || 60;
+    this.fuelConsumption = fuelConsumption || 10; 
+}
+var сurrentYear = new Date().getFullYear();
+
+Car.prototype.getFullName = function () {
+    console.log(this.name + " " + this.model);
+}
+var fullName = new Car("Mazda", 6);
```

Comment:
when you create an instance of `Car` constructor we receive an object of a car with set of properties (name, model, year and others are among of them) => therefore it's not right to name this variable as `fullName`